source/includes/common/common-install-operator-kubectl-plugin.rst

```diff
@@ -27,6 +27,8 @@ The command initializes the MinIO Operator with the following default settings:
   different :kube-docs:`cluster domain 
   <tasks/administer-cluster/dns-custom-nameservers/>` value.
 
+- Note that custom deployments are required to edit YAML manually to fulfill certain requirements or limitations such as those on resource requests.
```

Comment:
From reading the issue, it looks like the main issue is that if you have any custom requirements, you have to output the YAML and edit that, vs using the CLI alone.

Suggested re-write:

```suggestion

The Operator deploys with certain default settings and resource requests.
To modify these settings, do the following:

1. Append the ``kubectl minio init -o > operator.yaml`` to save the YAML configuration to file
2. Modify settings as-needed to fit your deployment
3. Run ``kubectul apply -f operator.yaml`` to apply the customized Operator deployment.
4. ```